OCP-Infrastructure/bootstrap/bootstrap.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
General comment: There are many many shell scripts? Is it really necessary to script so much for OCP4?

Reply:
No the amount of scripts does not have any correlation to OCP4.

The amount of scripts increased to provision each component of hogarama individual via bash.
Once we have helm as default method in place we can reduce the amount of bash scripts.

Reply:
the functions folder contains bash functions to store commonly used functions in order to avoid having each function defined in every bash script. By doing so we have a central place to alter commonly used functionality across our bash scripts.

Reply:
In OCP3 we have much less shell-scripting, still we can provision each component individually.